cypress.json

```diff
@@ -1 +1,5 @@
-{}
+{
+  "video": true,
+  "screenshotOnRunFailure": false,
+  "projectId": "97nm7q"
```

Comment:
is this property needed?

Reply:
Yes because I integrated the cypress dashboard

Reply:
Oh ok. nice :) can you add an screenshot of how looks like the dashboard

Reply:
Sure Here you can see all the runs that I did for this PR 
![Screen Shot 2022-06-03 at 11 41 30 AM](https://user-images.githubusercontent.com/7946622/171908938-a7a974d9-d3b3-457a-90c0-96a339b911a8.png)


And here You can see the results for one of the runs, you can see a video there too
![Screen Shot 2022-06-03 at 11 41 46 AM](https://user-images.githubusercontent.com/7946622/171908901-ef5e7ac9-4c1d-4cb7-bb72-56cbf101f4d2.png)

Reply:
I think you can access the project because is public, I'll leave here the link to it: https://dashboard.cypress.io/projects/97nm7q